src/cuga/backend/server/manage_routes/config_routes.py

```
@@ -0,0 +1,437 @@
+        state.config_version = ver
+        state.tools_include_version = int(ver) if ver else 0
+
+        if prepared_knowledge:
+            knowledge_result = engine.commit_knowledge_update(prepared_knowledge)
+
+        config.pop("knowledge_state", None)
+        await apply_published_config(state, config or {})
+
+        try:
+            await rebuild_production_agent(state, config or {})
+        except Exception as rebuild_err:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)